src/app/component/ubs/components/personal-data-form/add-address/add-address/add-address.component.html

```diff
@@ -0,0 +1,71 @@
+<form [formGroup]="addAddressForm" class="row adress">
+  <div class="col-12 col-sm-6 form-group">
+    <label>Місто</label>
+    <select formControlName="city" class="shadow-none form-control">
+      <option value="Kyiv">Київ</option>
+    </select>
+    <div class="w-100 city-notice">
+      На даний момент обслуговуємо лише м.Київ
+    </div>
+  </div>
+  <div class="col-12 col-sm-6 form-group">
+    <label>Район</label>
+    <input
+      formControlName="district"
+      class="shadow-none form-control"
+      matGoogleMapsAutocomplete
+      [country]="'ua'"
+      [types]="['geocode']"
+      (onAutocompleteSelected)="onDistrictSelected($event)"
+      [attr.disabled]="districtDisabled ? true : null"
+      [value]="region">
+    <div class="validation" *ngIf="addAddressForm.get('district').invalid && addAddressForm.get('district').touched">
```

Comment:
use class getter instead of addAddressForm.get('district')
get district() {
return this.addAddressForm.get('district')
}